src/apps/Agreement/AgreementDoc.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Box, textStyle, GU } from '@aragon/ui'
+import Markdown from '../../components/Markdown/Markdown'
```

Comment:
Now that aragonUI has a markdown component, let's switch over and replace this component with that one!

We may need separate ones for the AppCenter and Agreement UI to apply different styling.

Reply:
Good idea, I'll copy the existing styles over for now and we can make changes to them when required.